### PR TITLE
mail/postfix: add description to restart action

### LINKED ISSUE #1874: Postfix restart when certificate is updated by Lets Encrypt
Using Postfix/Rspamd/redis/clamav as mail gateway with a Lets Encrypt certificate for Postfix.

Need a hook to restart Postfix when acme client renew the Lets Encrypt certificate.

### PR DESCRIPTION
**Pull request by fraenki**
closes #1874

### CODE REVIEW

File: mail/postfix/Makefile [hunk 1]
```diff
@@ -1,5 +1,6 @@
 PLUGIN_NAME=		postfix
 PLUGIN_VERSION=		1.14
+PLUGIN_REVISION=  1
```
Comment (fichtner) [inline mail/postfix/Makefile:3]:
```suggestion
PLUGIN_REVISION= 	 	1
```

File: mail/postfix/src/opnsense/service/conf/actions.d/actions_postfix.conf [hunk 2]
```diff
@@ -15,6 +15,7 @@ command:/usr/local/opnsense/scripts/OPNsense/Postfix/setup.sh;/usr/local/etc/rc.
 parameters:
 type:script
 message:restarting Postfix
+description:Restart Postix service
```
Comment (fichtner) [inline mail/postfix/src/opnsense/service/conf/actions.d/actions_postfix.conf:18]:
```suggestion
description:Restart Postfix service
```

**Comment by fraenki** [comment]
uh-oh, so many issues in two lines of changes – thx for reviewing it

**Comment by fichtner** [comment]
No worries, only cosmetics :)

**Pull request merged**